src/pages/Admin/Admin_v2.tsx

```diff
@@ -23,6 +23,7 @@ import { orderedVenuesSelector } from "utils/selectors";
 import { useAdminVenues } from "hooks/useAdminVenues";
 import EntranceExperience from "./EntranceExperience";
 import AdvancedSettings from "./AdvancedSettings";
+import { WithNavigationBar } from "components/organisms/WithNavigationBar";
 // import TicketingAndAccess from "./TicketingAndAccess";
```

Comment:
[OOS] but is there any reason that this still needs to be here?

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/1181/commits/5beb20726bc5a1c35b173a916b2dc0e08910dc8b